framework/base/ErrorHandler.php

```diff
@@ -196,7 +196,7 @@ public function handleHhvmError($code, $message, $file, $line, $context, $backtr
      * @param string $message the error message.
      * @param string $file the filename that the error was raised in.
      * @param int $line the line number the error was raised at.
-     * @return bool whether the normal error handler continues.
+     * @return false whether the normal error handler continues.
```

Comment:
`handleError()` returns only `false`, which makes this return value useless. Same as this check: https://github.com/rob006/yii2-dev/blob/450eefaa50fc8aae1f9137a6fdf7ed2e5e9fe9e1/framework/base/ErrorHandler.php#L177

Am I missing something?